From: =?UTF-8?q?Carl=20F=C3=BCrstenberg?= <carl@excito.com>
Date: Mon, 21 Jan 2013 15:38:53 +0100
Subject: Fix shebang

---
 audiotools/player.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/audiotools/player.py b/audiotools/player.py
index ee05aa8..ce9085c 100644
--- a/audiotools/player.py
+++ b/audiotools/player.py
@@ -1,4 +1,4 @@
-#!/usr/bin/bin
+#!/usr/bin/python
 
 #Audio Tools, a module and set of tools for manipulating audio data
 #Copyright (C) 2007-2011  Brian Langenberger
-- 
